shared/constants/transaction.ts

```diff
@@ -284,12 +284,16 @@ export interface TxParams {
   accessList?: AccessList;
   maxFeePerGas?: string;
   maxPriorityFeePerGas?: string;
+  estimateSuggested?: string;
```

Comment:
Were these discovered as they were being read by the metric logic?

Should we try add comments to any new properties we add for future clarity?